backend/routes.py

```diff
@@ -4,26 +4,115 @@
 
 
 # Import all the necessary modules and packages
-from flask import current_app, request, jsonify, make_response
+# from flask import abort
+import jwt
+import uuid
+from flask import current_app, json, request, jsonify, make_response, abort
 from app import create_app, db
 from models import Login, User, Project
-
+from functools import wraps
+from werkzeug.security import generate_password_hash, check_password_hash
+# import werkzeug
 
 # Create an application instance
 app = create_app()
 
 
+def token_required(f):
+    @wraps(f)
+    def decorator(*args, **kwargs):
+
+        token = None
+
+        if 'x-access-tokens' in request.headers:
```

Comment:
What about included a 'Bearer' token too?

Reply:
Can you specify? Like should I have both access tokens and bearer token in the request / payload?

Reply:
I think Bearer tokens are better since they mostly the ones I've seen being used for OAuth systems
But I might be wrong.

Take this as a mild suggestion. #heurisitcs